modin/pandas/test/dataframe/test_iter.py

```diff
@@ -258,6 +258,21 @@ def test___setattr__():
     df_equals(modin_df, pandas_df)
 
 
+def test___setattr___getattr__():
```

Comment:
Could you please rename the previous test case, `test___setattr__`, to `test___setattr__not_column`? Also please add a note to that test case saying that it's testing setting an attribute that is not an existing column label.